cgmanifest.json

```diff
@@ -0,0 +1,3405 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "play-services-ads",
+          "GroupId": "com.google.android.gms",
+          "Version": "20.6.0",
+          "NuGetId": "Xamarin.GooglePlayServices.Ads",
+          "NuGetVersion": "120.6.0.1"
+        }
+      },
+      "License": "MIT",
+      "Description": "GPS-FB-MLKit and dependencies",
+      "LicenseDetail": [
+        "MIT by Microsoft, other by Google and other 3rd party"
+      ]
+    },
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "play-services-ads-base",
+          "GroupId": "com.google.android.gms",
+          "Version": "20.6.0",
+          "NuGetId": "Xamarin.GooglePlayServices.Ads.Base",
```

Comment:
> Should `NuGetId` and `NuGetVersion` be included? They are not in the existing `cgmanifest.json` files that are being removed.

https://github.com/xamarin/XamarinComponents/blob/a89581fd57351fc4fdbdebcf6548cc387f0be995/Android/CoilBase/cgmanifest.json

https://github.com/xamarin/XamarinComponents/blob/b613ea8f070338c575a1587a21d0d78e54ae4ca6/Android/GoogleDagger/cgmanifest.json

you approved

https://github.com/xamarin/XamarinComponents/pull/1025/files

https://github.com/xamarin/XamarinComponents/pull/1025/files

I will stop here.

If you think about it, the purpose of CG is to track buggy/insecure/faulty 3rd party artifacts. 

If some artifact+version is bound and delivered/published to our clients/customers it wold be good to know which nuget was that... And that would be NuGetId+Version.

Reply:
> Looking at the docs I could find (https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest), these are not valid values. Additionally, none of the license values are specified. Do you have a more recent doc that describes the structure of the `cgmanifest.json` file?

https://github.com/microsoft/vscode/blob/main/.vscode/cgmanifest.schema.json

Why is it a problem to add NugetId and NugetVersion?

Reply:
if you see here

no one thinks about Maven inside Nuget, so there is no data model for that

https://github.com/marklindsey11/PowerShell/commit/01069f03fc78c588f06571d67bad297d3649f796

maven inside nuget is bindings we do

Reply:
> If you think about it, the purpose of CG is to track buggy/insecure/faulty 3rd party artifacts.

This is true, but this is accomplished by having a contract that standard tools can parse and understand.  Adding our own non-standard data to it will not benefit anyone because no tools will be able to understand it.

Reply:
`NugetId` was redth's idea for improvements (maven to artifact mappings). He added it to several bindings in XamarinComponents repo and he used it in `Xamarin.Binding.Helpers`. I have no details about the project.

https://github.com/Redth/Xamarin.Binding.Helpers

I found the schema after I generated the code.

Reply:
removed all extras, which were old attempts to solve problems (talked to redth about it).